tests/test.linter.js

```diff
@@ -1253,6 +1252,41 @@ describe('Linter.extractMetadata()', function() {
       });
   });
 
+  // Total zip size is 96080 but only a handful of files are actually
+  // scanned.
+
+  // Archive:  tests/fixtures/empty-with-library.zip
+  // Length   Date       Time   Name
+  // -------  ---------- -----  ----
+  //   593    2015-11-28 19:46  bootstrap.js -> scanned
+  //     0    2017-05-09 15:09  data/
+  //  6148    2017-05-09 15:09  data/.DS_Store -> scanned (binary)
+  //     0    2017-05-09 15:09  __MACOSX/
+  //     0    2017-05-09 15:09  __MACOSX/data/
+  //   120    2017-05-09 15:09  __MACOSX/data/._.DS_Store -> scanned
+  //  1420    2015-11-28 19:46  data/change-text.js -> scanned
+  //     0    2015-11-28 19:46  data/empty.js -> scanned
+  // 86659    2017-03-20 20:01  data/jquery-3.2.1.min.js
```

Comment:
Nit: maybe we could add an explicit `SKIPPED` or something? I believe that would make the test easier to understand, especially how you get from 96080 to 9421.